recipes/cereal/bld.bat

```diff
@@ -0,0 +1 @@
+xcopy /s /f include %LIBRARY_INC%
```

Comment:
`noarch: generic` package builds are just run on Linux, no need for this file.

Reply:
So, the CI will fail here and that's not a problem then.